apps/docs/src/app/core/component-docs/shellbar/examples/shellbar-side-nav/shellbar-side-nav-example.component.html

```diff
@@ -9,32 +9,32 @@
     ></button>
 
     <fd-shellbar-logo>
-        <a href="#" class="fd-shellbar__logo fd-shellbar__logo--image-replaced" aria-label="SAP"></a>
+        <a href="#" class="fd-shellbar__logo fd-shellbar__logo--image-replaced" alt="SAP Logo"></a>
```

Comment:
I know it was like this before this PR but this should be a component or a directive. The `fd-shellbar__logo--image-replaced` class is just bringing the SAP logo so it can be a boolean. User should be able to pass any svg and alt text.

Reply:
@InnaAtanasova this fd-shellbar__logo--image-replaced is already hardcoded with the sap Logo so its not necessary to create a directive or a component, this shows an example how the user can create his own css class and add it to the logo component
<img width="412" alt="Screenshot 2022-01-06 at 12 38 23 PM" src="https://user-images.githubusercontent.com/53534379/148343193-6bf6627d-2fbf-4b80-9ab4-cb915b5ed98d.png">
.

Reply:
I'm sorry, I don't get it. `fd-shellbar__logo` and `fd-shellbar__logo--image-replaced` are classnames prefixed with `fd-`, indicating they are fundamental classes. How are they user defined classes?
`fd-shellbar__logo` is a class coming from Fundamental-styles, so the user should not be setting an HTML element with this class. It should be a directive or a component. All Core components allow the users to add their custom classes with which they modify the properties coming from the library. If you want to illustrate such example, please use an <a> element with a logo directive that will bring the  `fd-shellbar__logo` class and then add a custom class that is not starting with `fd-` and provide an explanation in the description above. If that's not the case, please provide more info, coz I really don't get it. Thank you!

Reply:
https://github.com/SAP/fundamental-ngx/pull/7496/files#r782453420